src/chainstate/stacks/boot/pox.clar

```diff
@@ -467,3 +450,15 @@
 
     (ok true)))
 )
+
+;; Used for PoX parameters discovery
+(define-read-only (get-pox-info)
+    (ok {
+        min-amount-ustx: (get-stacking-minimum),
+        reward-cycle-id: (current-pox-reward-cycle),
+        registration-window-length: (var-get pox-registration-window-length),
```

Comment:
I think we should stop using `registration-window-length` -- everywhere but `pox.clar` uses prepare length.

Reply:
Agreed.  This is my bad -- I think I was the one who erroneously referred to the prepare phase as a "registration window."

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1899/commits/68dedfb374c9e8923e3ea34f14c662032f1fe67b